### ISSUE TITLE
New Tab Drop-down should be keyboard accessible

### ISSUE THREAD

**Issue by bitcrazed** [microsoft/terminal#2181]
# Description of the new feature/enhancement
Users should be able to display the "new tab drop-down" using a key chord.

I'd prefer <kbd>CTRL</kbd><kbd>SHIFT</kbd><kbd>SPACE</kbd> but I am sure others would prefer a different chord, so this chord-action should key-mappable.

**Comment by miniksa** [comment]
OK we should probably leave this default mapped for accessibility but then let people who don't want it to remove it.
